src/app/_services/api.service.ts

```diff
@@ -87,6 +88,25 @@ export class ApiService {
     }
   }
 
+  checkOidcToken(token: string): Observable<{ identity: Partial<IIdentity>; callbackUrl: string; }> {
+    this._getApiUrl();
+
+    const url = new URL(`${this.apiUrl}/oidc/token`);
```

Comment:
Why not directly create a string?
```suggestion
    const url = `${this.apiUrl}/oidc/token`;
```